source/scss/typography/_component-typography.scss

```diff
@@ -1,18 +1,35 @@
-// Global component typgraphy, that does not belong to normal flowing text
+////
```

Comment:
And what do four slashes mean? Beginning/end of documentation blocks?

Reply:
These comments, you will see in the beginning of the page. When writing documentation, you will notice, that some of the things you want to describe apply to all, or at least almost all of the items inside a file.

Here, you want to apply the first-level annotations, that automatically add annotations to all items, that have at least a description `/// foo bar description`, or any other annotation.

I started having the author as Planet Express using our email address. Some of the files are good to have the annotation @group.

First-level annotations must start with 4 slashes and must end with 4 slashes.
In between them, they can have any number from 3, 4 or 5.

Note: First-level annotations can be overwritten on annotations attached to the item.
```
////
/// @author Planet Express
/// @group typography
/// A description applied to all items in this file
////

/// A different item I didn't document, but another person did
$another-variable: red:

/// This is an item I want to document
/// @author Fillipe
/// @group color
$typography-variable: #333333;
```

Output
```
>> A different item I didn't document, but another person did
>> Author: Planet Express
>> Group: typography
$another-variable: red;

>> This is an item I want to document:
>> Author: Fillip
>> Group: color
$typography-variable: #333333;
```

[See First-Level annotations documentation](http://sassdoc.com/file-level-annotations/).